helper/include/odp/helper/ip.h

```diff
@@ -112,7 +112,8 @@ static inline int odph_ipv4_csum(odp_packet_t pkt,
 				 odp_u16sum_t *chksum)
 {
 	uint32_t nleft = (uint32_t)(ODPH_IPV4HDR_IHL(ip->ver_ihl) * 4);
-	uint16_t buf[nleft / 2];
+	const int max_ip_header_len = 15 * 4;
```

Comment:
15 * 4 could be useful as a public constant as well (e.g. `ODPH_IPV4HDR_MAX_LEN`).

Reply:
As per offline discussions, leaving this as is to avoid public changes.